src/routes/courses.js

```diff
@@ -16,7 +16,10 @@ router.get('/:course_id', (req, res) => {
     res.json(courseloader.getCourseById(req.params.course_id));
   } else {
     res.status(404);
```

Comment:
Just realized there's a duplicate status 404......Stupid of me not noticing this on first sight, maybe remove the res.status(404) statement

Reply:
what do you mean? Why don't we need to specify the return status when things go wrong? 404 represent page not found. Shall we use 406 in this case to indicate that if there is no such course found with the given course ID from the client?